extensions/wikia/ArticleVideoMVP/ArticleVideoMVP.setup.php

```diff
@@ -0,0 +1,19 @@
+<?php
+$wgAutoloadClasses['ArticleVideoMVPHooks'] = __DIR__ . '/ArticleVideoMVP.hooks.php';
+$wgAutoloadClasses['ArticleVideoMVPController'] = __DIR__ . '/ArticleVideoMVPController.class.php';
+
+$wgHooks['BeforePageDisplay'][] = 'ArticleVideoMVPHooks::onBeforePageDisplay';
+$wgHooks['MakeGlobalVariablesScript'][] = 'ArticleVideoMVPHooks::onMakeGlobalVariablesScript';
+
+
+// cityId => articleId => videoDetails
+$wgVideoMVPArticles = [
```

Comment:
If you move it to wikifactory then you can avoid one level in this array. WF can set wg variables per wiki.